As we don't have SVG support we disable the search for the converter.

Author: Alexander Grund (TU Dresden)
Updated for v44.0 by Simon Branford (University of Birmingham)

diff -aur a/configure.ac b/configure.ac
--- configure.ac.orig	2023-07-24 21:38:28.066689617 +0100
+++ configure.ac	2023-07-24 21:38:54.077313220 +0100
@@ -49,7 +49,7 @@
 AM_CONDITIONAL(ALLOW_RENDERING, test "x$allow_rendering" = "xyes")
 
 symbolic_encode_sizes="16x16 32x32"
-AC_PATH_PROG([GTK_ENCODE_SYMBOLIC_SVG], [gtk-encode-symbolic-svg], [false])
+GTK_ENCODE_SYMBOLIC_SVG="false"
 if test "x$GTK_ENCODE_SYMBOLIC_SVG" = "xfalse"; then
    symbolic_encode_sizes=""
 fi
